WcaOnRails/app/models/competition.rb

```diff
@@ -824,6 +824,25 @@ def to_wcif
     }
   end
 
+  def load_wcif!(wcif)
```

Comment:
Personally I never us "load" in names, since it's ambiguous - used both for upload and download.

I expected this function to store a WCIF JSON document to the DB, from the name.

Maybe `set_events!(events_wcif)`?